src/frontend/src/app/app-routing.module.ts

```diff
@@ -13,16 +12,12 @@ import { UsersComponent } from './components/users/users.component';
 const routes: Routes = [
 
   { path: '', redirectTo: 'home', pathMatch: 'full' },
-  { path: '',
-    component: LayoutComponent,
-    children: [
-      { path: 'login', component: LoginComponent},
-      { path: 'register', component: RegisterComponent},
-      { path: 'home', component: HomeComponent },
-      { path: 'notauthorized', component: NotauthorizedComponent},
-      { path: 'users', component: UsersComponent, canActivate: [AuthGuard]},
-    ]},
-{ path: '**', component: NotfoundComponent },
+  { path: 'login', component: LoginComponent},
+  { path: 'register', component: RegisterComponent},
+  { path: 'home', component: HomeComponent },
+  { path: 'notauthorized', component: NotauthorizedComponent},
+  { path: 'users', component: UsersComponent, canActivate: [AuthGuard]},
+  { path: '**', component: NotfoundComponent },
```

Comment:
I'm curious why routing is changing this drastically for adding a search bar. 🤔

Reply:
I was having an issue at one point, got confused, and changed this. 
I don't know why we have a 'LayoutComponent' and only a `router-outlet` in the app.component.html. 

If there is a good reason, then by all means let's keep it, but it is confusing to me.